cardano-cli/src/Cardano/CLI/EraBased/Common/Option.hs

```diff
@@ -1803,26 +1805,44 @@ make' format desc flag_ extraHelp kind =
       , Opt.long ("output-" <> flag_)
       ]
 
-pFormatCBOR
-  :: FormatCBOR :| fs
-  => String
+parserFromFormatFlags
+  :: String
+  -> [Flag (Vary fs)]
   -> Parser (Vary fs)
-pFormatCBOR =
-  make' FormatCBOR "BASE16 CBOR" "cbor" Nothing
+parserFromFormatFlags content =
+  parserFromFlags $ \f ->
+    mconcat
+      [ "Format "
+      , content
+      , " to "
+      , f.format
+      , if f.options.isDefault then " (default)" else ""
+      , "."
+      ]
```

Comment:
We can still have flexibility on the help for the flag by providing a rendering function like this.